crates/ra_project_model/src/sysroot.rs

```diff
@@ -100,9 +99,15 @@ fn try_find_src_path(cargo_toml: &Path) -> Result<PathBuf> {
         .current_dir(cargo_toml.parent().unwrap())
         .args(&["--print", "sysroot"])
         .output()
-        .map_err(|e| format!("rustc --print sysroot failed: {}", e))?;
+        .map_err(|e| anyhow!("rustc --print sysroot failed: {}", e))?;
```

Comment:
Here `.context` would be more appropriate I think

Reply:
done